kiali-server/templates/deployment.yaml

```
@@ -91,6 +89,10 @@ spec:
           valueFrom:
             fieldRef:
               fieldPath: metadata.namespace
+        - name: LOG_LEVEL
+          value: {{ include "kiali-server.logLevel" . }}
```

should this value be wrapped in quotes?
```suggestion
          value: "{{ include "kiali-server.logLevel" . }}"
```